### ISSUE TITLE
create a point with gps position #enhancement

### ISSUE THREAD

**Issue by jancelin** [3liz/lizmap-web-client#187]
is it possible to create a point with the GPS position?

**Comment by rldhont** [comment]
This feature has been added in this PR #693
@jancelin can you review it ?

**Comment by jancelin** [comment]
Whouaou !!!!

Works fine for me, just the "Après enregistrement / Fermer le formulaire" that does not close the form.
I'm testing out at lunch. 

Result here:
https://demo.sig.inra.fr/websig/lizmap/www/index.php/view/map/?repository=demo&project=test

I put your master-edition-32 branch in docker container testing: https://github.com/jancelin/docker-lizmap/blob/testing/Dockerfile

**Comment by jancelin** [comment]
The geolocation in editing works well, but is difficult to use on smartphone because SCR X Y takes up too much space. On a 10-inch shelf it's fine.

bug drop-down menu:
Loss of drop-down lists in the edition. 
The automatic sowing input works with the keyboard but not with touch.

**Comment by josemvm** [comment]
https://github.com/3liz/lizmap-web-client/issues/638

**Comment by rldhont** [comment]
@jancelin what are the logs for :  just the "Après enregistrement / Fermer le formulaire" that does not close the form.

**Comment by jancelin** [comment]
the 3 possibilities in "After registration" always do the same job:"create a new object".
log:
```
2017-12-05 14:08:59	138.102.212.90	warning	2017-12-05 14:08:59	[2]	Invalid argument supplied for foreach()	/var/www/websig/lizmap/modules/lizmap/classes/qgisVectorLayer.class.php	416

2017-12-05 14:08:59	138.102.212.90	notice	2017-12-05 14:08:59	[8]	Undefined index: logEmail	/var/www/websig/lizmap/modules/lizmap/classes/lizmapLogItem.class.php	69

2017-12-05 14:08:59	138.102.212.90	notice	2017-12-05 14:08:59	[8]	Undefined index: logEmail	/var/www/websig/lizmap/modules/lizmap/classes/lizmapLogItem.class.php	69
```

**Comment by rldhont** [comment]
Hi @jancelin I have fix the issue with PG Feature insert and update with close form 10ac1116bbf7bfbc4cd067b23adc8e719004b129

**Comment by rldhont** [comment]
fix for mobile form 9d2d5a553eed6a5af64decfa86b10596899303a3
